### PR TITLE
User models and registration

### PR DESCRIPTION
**Pull request by nicoglennon**
review time!!!! coo coo

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+
```
Comment (kvrag) [inline app/controllers/sessions_controller.rb:2]: We need a new session action (log in form?) here as well. Maybe we can clean up the p and puts debugging statements.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      puts "It saved!!"
```
Comment (kvrag) [inline app/controllers/users_controller.rb]: clean up puts

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  has_secure_password
```
Comment (kvrag) [inline app/models/user.rb:2]: add user validations: email uniqueness, password, etc. also associations

**Review by kvrag — CHANGES_REQUESTED** [review]
see comments

**Comment by ehickle** [comment]
Cleaned up put statements. Resubmit pull request with noted changes.

**Review by kvrag — APPROVED** [review]

**Pull request merged**